.github/workflows/ci_rust.yml

```diff
@@ -59,6 +59,29 @@ jobs:
           ./generate.sh
           ldd target/debug/integration | grep libs2n.so
 
+  # our benchmark testing includes interop tests between s2n-tls, rustls, and
+  # openssl
+  bench-tests:
```

Comment:
I mentioned this concern a while back, but you might have forgotten: if I see the name "bench / benchmark tests", I don't assume interop testing. I assume performance benchmarks. Does this actually run any performance benchmarks? Or is it just running interop tests?

Reply:
https://github.com/aws/s2n-tls/blob/main/bindings/rust/bench/src/harness.rs#L426

It runs these tests, which for all of OpenSSL, Rustls, s2n-tls, will test the product of 
- handshake type
- cipher suite
- kx type
- cert type

Also session resumption interop tests.

And then checks that the certificate file paths are valid.

So this doesn't do any performance benchmarks.

Do you have any name suggestions? I was just sticking to the lowest denominator in that these are literally the tests in the bench crate. The longer term plan that has been tossed around is that we will shift all of our test harnesses to use this, at which point these tests could live in the `integration` crate?